data/SJ-Kumar.json

```diff
@@ -0,0 +1,48 @@
+{
+"name": "Jayanth Kumar",
+  "type": "personal",
+  "bio": "Open Source DevRel |CSE Student | **GitHub** Star",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "DevOps",
+    "DevRel"
+  ],
+  "socials": [
+    { "icon": "FaLinkedIn", "url": "https://www.linkedin.com/in/sjayanthkumar/"}, 
+    { "icon": "FaTwitter", "url": "https://twitter.com/Jayanth_2003" },
+    { "icon": "FaGithub", "url": "https://github.com/SJ-Kumar" }
+    
+  ],
+  "testimonials": [
+    "CBID2",
+    "Akshu-on-github",
+    "ZeeMonk-pixel",
+    "krupalitrivedi"
+  ],
```

Comment:
You might want to remove this section as you don't have any testimonials yet ig. 

```suggestion
```

Reply:
Sure, for now, I will remove. After I gather all the required details. I will add this section.